nodes/index.adoc

```diff
@@ -17,6 +17,18 @@ Having stable and healthy nodes in a cluster is fundamental to the smooth functi
 In {product-title}, you can access, manage, and monitor a node through the `Node` object representing the node.
 Using the OpenShift CLI (`oc`) or the web console, you can perform the following operations on a node.
 
+The following components of a node are responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
+
+* *Container runtime*: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
+
+* *Kubelet*: Kubelet runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running. The kubelet process maintains the state of work and the node server. Kubelet manages network rules and port forwarding. The kubelet manages containers that are created by Kubernetes only.
```

Comment:
```suggestion
* *Kubelet*: Kubelet runs on nodes and reads the container manifests and ensures that the defined containers have started and are running. The kubelet process maintains the state of work and the node server. Kubelet manages network rules and port forwarding. The kubelet manages containers that are created by Kubernetes only.
```

Reply:
Ensures that the defined containers... is a fragment. You can either join it with the other sentence or use "It".

Reply:
Addressed